### ISSUE TITLE
Cannot find module - Vue3, Vite & Vitest

### ISSUE THREAD

**Issue by mikecbone** [fullcalendar/fullcalendar#7125]
### Reduced Test Case

https://stackblitz.com/edit/github-c67sof?file=tests/CalendarFreeComponent.test.js&view=editor

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

Vue

### Bug Description

When running very basic tests using `vitest`, if a component imports `FullCalendar` then the test fails with 

```js
Error: Cannot find module '.../node_modules/@fullcalendar/vue3/dist/FullCalendar' imported from '.../node_modules/@fullcalendar/vue3/dist/index.js'
```

Attached test case demos this if you run `yarn test`.

**Comment by puppe82** [comment]
Maybe it's the same issue I was having with the new FullCalendar 6.x version.

Webpack 5 expects modules to have file extensions for `import` statements. In order to disable the default behavior, one needs to set a rule `fullySpecified: false`

---- 

### Webpack 5: Import fails to resolve with Module not found error
https://github.com/fullcalendar/fullcalendar/issues/7114

### webpack 5.0.0-beta.30 Module not found: Error: Can't resolve '../../core-js/object/define-property'
https://github.com/webpack/webpack/issues/11467

### Webpack documentation
https://webpack.js.org/configuration/module/#resolvefullyspecified

**Comment by arshaw** [comment]
Queued for release. See https://github.com/fullcalendar/fullcalendar/issues/7114#issuecomment-1379235816

**Comment by arshaw** [comment]
Released in [v6.0.3](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.0.3)

Updated repro:
https://stackblitz.com/edit/github-c67sof-bd165u?file=package.json&view=editor

![Screen Shot 2023-01-11 at 5 26 18 PM](https://user-images.githubusercontent.com/56520/211931270-4ab17c82-c474-40cc-8b94-069a2e4cc7a8.png)
